for_challenges.py

```diff
@@ -40,7 +46,12 @@
     ['Вася', 'Маша', 'Саша', 'Женя'],
     ['Оля', 'Петя', 'Гриша'],
 ]
-# ???
+print(f'Total grous: {len(groups)}')
+group_index = 1
+for group in groups:
```

Comment:
для итерации с индексом в питоне есть enumerate
```suggestion
for group_index, group in enumerate(groups):
```
если надо начать не с 0, а с 1, то так
```suggestion
for group_index, group in enumerate(groups, start=1):
```